src/popup/components/offerInfo.tsx

```diff
@@ -86,7 +86,7 @@ function offerInfo({ request }: IPopupPageProps<MethodEnum.REQUEST>) {
                                     {asset.assetId
                                         ? finsAsset?.name ||
                                           `CAT ${shortenHash(asset.assetId)}`
-                                        : XCH.code}
+                                        : 'XCH'}
```

Comment:
no 'TXCH' ?

Reply:
because  AssetIcon components use XCH code for xch icon (https://github.com/hashgreen/hoogii-wallet/blob/main/src/components/AssetIcon.tsx#L15)